src/main/java/org/sqlite/Function.java

```diff
@@ -141,7 +137,7 @@ public static final void destroy(Connection conn, String name, int nArgs) throws
      * @param name The name of the function.
      * @throws SQLException
      */
-    public static final void destroy(Connection conn, String name) throws SQLException {
```

Comment:
why remove the `final` here?

Reply:
I removed this change. This change was done because intellij thinks that declaring static methods as final is useless, as polymorphism isn't possible with static methods.

Looking into this a bit more, it seems that it does actually do something - if you extend the class Function, it will prohibit you from making a method on it called destroy, static or otherwise.

I think this is a stylistic choice, and therefore keeping the style of the project makes sense. And if the style was to change, it certainly shouldn't be pushed in as part of a bugfix.